### PR TITLE
Lms student model

### PR DESCRIPTION
**Pull request by darkydash**

### CODE REVIEW

**Comment by darkydash** [comment]
```JSON
{
  "milgroup": {
    "milgroup": 0,
    "milfaculty": "string",
    "weekday": 0
  },
  "program": {
    "code": "string",
    "program": "string",
    "faculty": "string"
  },
  "recruitment_office": {
    "city": "string",
    "district": "string"
  },
  "university_info": {
    "id_card": "string",
    "group_title": "string"
  },
  "passport": {
    "passport_series": "string",
    "passport_code": "string",
    "passport_ufms_name": "string",
    "passport_ufms_code": "string",
    "passport_date": "2021-03-12"
  },
  "family": {
    "mother": {
      "surname": "string",
      "name": "string",
      "patronymic": "string",
      "rel_status": "father",
      "birthdate": "2021-03-12",
      "citizenship": "string",
      "pers_mobile": "string",
      "permanent_address": "string",
      "place_birth": "string"
    },
    "father": {
      "surname": "string",
      "name": "string",
      "patronymic": "string",
      "rel_status": "father",
      "birthdate": "2021-03-12",
      "citizenship": "string",
      "pers_mobile": "string",
      "permanent_address": "string",
      "place_birth": "string"
    },
    "brothers": [
      {
        "surname": "string",
        "name": "string",
        "patronymic": "string",
        "rel_status": "father",
        "birthdate": "2021-03-12",
        "citizenship": "string",
        "pers_mobile": "string",
        "permanent_address": "string",
        "place_birth": "string"
      }
    ],
    "sisters": [
      {
        "surname": "string",
        "name": "string",
        "patronymic": "string",
        "rel_status": "father",
        "birthdate": "2021-03-12",
        "citizenship": "string",
        "pers_mobile": "string",
        "permanent_address": "string",
        "place_birth": "string"
      }
    ]
  },
  "surname": "string",
  "name": "string",
  "patronymic": "string",
  "birthdate": "2021-03-12",
  "photo": "string",
  "place_birth": "string",
  "region_birth": "string",
  "citizenship": "string",
  "pers_mobile": "string",
  "corp_email": "user@example.com",
  "pers_email": "user@example.com",
  "permanent_address": "string",
  "surname_genitive": "string",
  "name_genitive": "string",
  "patronymic_genitive": "string",
  "mil_specialty": "string",
  "status": "student"
}
```

File: back-end/lms/models/student.py [hunk 1]
```diff
@@ -55,27 +55,25 @@ class Meta:
...
+    id_card = models.CharField(max_length=32)
+    group_title = models.CharField(max_length=32)
+
+
+class Passport(models.Model):
     passport_series = models.CharField(max_length=4)
     passport_code = models.CharField(max_length=6)
     passport_ufms_name = models.CharField(max_length=255)
     passport_ufms_code = models.CharField(max_length=7)
     passport_date = models.DateField()
```
Comment (TmLev) [inline back-end/lms/models/student.py:73]: `issue_date`

Comment (TmLev) [inline back-end/lms/models/student.py:73]: Remove `passport_` prefix.

File: back-end/lms/models/student.py [hunk 2]
```diff
@@ -101,11 +99,28 @@ class Student(Personnel):
...
     name_genitive = models.CharField(max_length=32)
     patronymic_genitive = models.CharField(max_length=32, blank=True)
 
+    milgroup = models.ForeignKey(Milgroup, models.DO_NOTHING)
+    mil_specialty = models.ForeignKey(MilSpecialty, models.DO_NOTHING)
+
+    program = models.ForeignKey(Program, models.DO_NOTHING)
+    status = models.ForeignKey(Status, models.DO_NOTHING)
+
+    passport = models.ForeignKey(Passport, models.DO_NOTHING)
```
Comment (TmLev) [inline back-end/lms/models/student.py:114]: `OneToOne`

**Comment by TmLev** [comment]
Surpassed by #221.

**Pull request closed without merging**